pywinauto/hooks.py

```diff
@@ -0,0 +1,231 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_void_p, byref
+import atexit
+
+
+def create_pointer(handler):
```

Comment:
`_create_callback_pointer` or just `_callback_pointer` would be more informative. It seems we don't need to share this function with other users so let's keep it private.